recipes/scikit-downscale/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scikit-downscale" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07bcfb38ef7b53004cdaba6d93d6d771795c3fee7ba4e311025c1a42af38a996
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - xarray
```

Comment:
My only question is if we need to reproduce the version restrictions on `xrray` and `scikit-learn` that are in the `requirements.txt` file upstream:

https://github.com/jhamman/scikit-downscale/blob/master/requirements.txt#L1-L2

Reply:
Thanks for the review @synapticarbors. I went ahead and added the version restrictions.